Dockerfile.aarch64

```diff
@@ -23,7 +23,7 @@
 # the case. Therefore, you don't have to disable it anymore.
 #
 
-FROM aarch64/ubuntu:trusty
+FROM aarch64/ubuntu:vivid
```

Comment:
In addition, we shouldn't use ubuntu:vivid, because it's EOL, and we no longer build binaries for that, so better not use it in our Dockerfiles as well. ubuntu:trusty (14.04) is a LTS version.